libraries/recipe_helpers.rb

```diff
@@ -98,6 +98,25 @@ def cookbook_version(run_context)
         run_context.cookbook_collection['datadog'].version
       end
 
+      def upstart_platform?(node)
+        agent_major_version(node) > 5 &&
+          (((node['platform'] == 'amazon' || node['platform_family'] == 'amazon') && node['platform_version'].to_i != 2) ||
+           (node['platform'] == 'ubuntu' && node['platform_version'].to_f < 15.04) || # chef <11.14 doesn't use the correct service provider
+          (node['platform'] != 'amazon' && node['platform_family'] == 'rhel' && node['platform_version'].to_i < 7))
```

Comment:
Why do we have to use `platform_family?` elsewhere but we use `node['platform_family']` here?

Reply:
For some reason, the method is not defined from the `recipe_helpers.rb`. I didn't investigate why.

Reply:
🤔 Okay, it should work the same way so we can leave like that I guess